graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/NodeHelper.java

```diff
@@ -163,10 +177,18 @@ public static JCRNodeWrapper getNodeInLanguage(JCRNodeWrapper node, String langu
             return node;
         }
         Locale locale = LanguageCodeConverters.languageCodeToLocale(language);
-        JCRSessionWrapper session = JCRSessionFactory.getInstance().getCurrentUserSession(workspace, locale);
+        JCRSessionWrapper session = JCRSessionFactory.getInstance().getCurrentUserSession(workspace, locale,
+                useFallbackLanguage  ? getFallbackLanguage(node): null);
         return session.getNodeByIdentifier(node.getIdentifier());
     }
 
+    private static Locale getFallbackLanguage(JCRNodeWrapper node) throws RepositoryException {
+        if (node.getResolveSite().isMixLanguagesActive()) {
```

Comment:
De we want to add safety check on `node.getResolveSite()` being null in case people ask for it on non site nodes ? 
Normally there should be a fallback on `systemSite`, but just in case something goes wrong it could return `null`